java/src/jmri/jmrix/loconet/locobuffer/LocoBufferAdapter.java

```diff
@@ -78,6 +78,7 @@ public String openPort(String portName, String appName) {
         setBaudRate(currentSerialPort, baud);
         configureLeads(currentSerialPort, true, true);
         setLocalFlowControl();
+        serialPort.setComPortTimeouts(com.fazecast.jSerialComm.SerialPort.TIMEOUT_READ_SEMI_BLOCKING, 100, 0);
```

Comment:
Not sure, but it seems that `serialPort` should be `currentSerialPort`.

Reply:
Fixed. Sorry for the problem.

Reply:
No problem! Happy to help!